packages/js-moi-providers/__tests__/ws-provider.test.ts

```diff
@@ -1,91 +1,159 @@
-import { Signer } from "js-moi-signer";
-import { AssetStandard, hexToBN, IxType } from "js-moi-utils";
-import { JsonRpcProvider } from "../src.ts/jsonrpc-provider";
-import { WebSocketEvents, WebSocketProvider } from "../src.ts/websocket-provider";
-import { initializeWallet } from "./utils/utils";
+import { hexToBytes, IxType } from "js-moi-utils";
+import { Wallet } from "js-moi-wallet";
+import { WebSocketEvent } from "../src.ts/websocket-events";
+import { WebsocketProvider } from "../src.ts/websocket-provider";
+import { getRandomSupply, initializeWallet } from "./utils/utils";
+
+const HOST = "<YOUR WEBSOCKET HOST PATH (ws://localhost:1600/ws)>";
+const MNEMONIC = "<YOUR SEED RECOVERY PHRASE>";
+const ASSET_ID = "0x00000000c67fbf5825d505586701135114fe76f5c0b18675c4a08467e8aa1495482c20c7";
 
 describe("Test Websocket Provider", () => {
```

Comment:
This has to be cleaned

Reply:
https://github.com/sarvalabs/js-moi-sdk/pull/91/commits/18386312549fcd592a23670c5e5ddf5dce441b29

Reply:
[ec3d40d](https://github.com/sarvalabs/js-moi-sdk/pull/91/commits/ec3d40d8728d2616dfe3cec985619835dd679193)

To prevent test cases from running if the setup fails, I implemented assertions using `assert` which is node package.  Initially, I attempted to use `expect` statements for validation; however, these caused the tests to continue running despite failures in the setup phase. Therefore, I switched to assertions to ensure that any setup failure halts further test execution.